lib/satellite/jwt_user_decoder.rb

```diff
@@ -0,0 +1,24 @@
+require 'jwt'
+
+module Satellite
+  class JWTUserDecoder
+    attr_reader :token
+
+    def initialize(token)
+      @token = token
+    end
+
+    def payload
+      return {} unless token.present?
+      @payload ||= ::JWT.decode(token, Satellite.configuration.jwt_secret_key_base).first
+    end
+
+    def user
+      ::User.find_by(uid: user_uid)
+    end
+
+    def user_uid
+      payload["id"]
```

Comment:
Why? `{}["id"] => nil`. `User.find_by(uid: nil)` also doesnt raise an error. Is code outside of this class relying on the user_id being present?